### PR TITLE
Clear state on decrypt password page issue77

### PR DESCRIPTION
**Pull request by SumitKumar-17**
### Clear state on decrypt password page  with reference to issue 77

To clear the previous decrypted data from state when the component mounts, you can add a cleanup function in the useEffect hook with an empty dependency array []. Here's how you can modify your useEffect hook:By dispatching decryptPasswordThunk({ passUid, pmsFile: null }), you're effectively clearing the decrypted data when the component mounts.

### CODE REVIEW

File: frontend/src/components/pages/DecryptPassword/DecryptPassword.js [hunk 1]
```diff
@@ -69,8 +69,10 @@ const DecryptPassword = () => {
   };
 
   useEffect(() => {
+    dispatch(decryptPasswordThunk({ passUid, pmsFile: null }));
```
Comment (frankhart2018) [inline frontend/src/components/pages/DecryptPassword/DecryptPassword.js:76]: Why does the API need to be called?

**Review by frankhart2018 — APPROVED** [review]

**Review by frankhart2018 — CHANGES_REQUESTED** [review]
Please make the required changes. s

**Review by frankhart2018 — CHANGES_REQUESTED** [review]
There are merge conflicts. Please resolve those.

**Comment by SumitKumar-17** [comment]
@frankhart2018  Completed the merge conflicts.

**Comment by frankhart2018** [comment]
Did you actually change anything? I still see a call to the thunk that calls the API. Clearing state does not have to require calling the API.

**Comment by SumitKumar-17** [comment]
@frankhart2018  No, I didn't change anything in my code as I gave you the first time. You had asked me about details of what I had done. I explained it above.
After that further, I had just resolved only merge conflicts due to the latest updates in the master branch

**Comment by frankhart2018** [comment]
> @frankhart2018 No, I didn't change anything in my code as I gave you the first time. You had asked me about details of what I had done. I explained it above. After that further, I had just resolved only merge conflicts due to the latest updates in the master branch

The question was rhetorical. The answer is no it does not need to call the API and hence does not need to call the thunk. Please fix that in a new PR, closing this.

**Pull request closed without merging**